API/MetAPI/ForecastDataType.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Backend.API.MetAPI.SubClasses;
+using GraphQL.Types;
+
+namespace Backend.API.MetAPI
+{
+    public sealed class ForecastDataType : ObjectGraphType<ForecastData>
+    {
+        public ForecastDataType()
+        {
+            Field(data => data.Instant, false, typeof(InstantType)).Description("Details about current data"); 
+            Field(data => data.Next1Hours, false, typeof(Next1HoursType)).Description("Summary and details(inconsistent data available)");
+            Field(data => data.Next6Hours, false, typeof(Next6HoursType)).Description("Summary and details(inconsistent data available)");
+            Field(data => data.Next12Hours, false, typeof(Next12HoursType)).Description("Summary and details(inconsistent data available)");
+            /*
+             * Instant does only contain Details, while the others also have a summary
+             */
```

Comment:
In the future, this can be added as [class documentation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments).

Don't worry about this now!